.github/workflows/go.yml

```diff
@@ -0,0 +1,27 @@
+name: Go
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.12
```

Comment:
why not 1.16?

Reply:
according to https://github.com/brancz/gojsontoyaml/blob/master/go.mod#L3

Reply:
oh, do you mind bumping both of those? then I'm happy to merge and create a tag immediately! :)